internal/operators/ocs/ocs_operator_test.go

```diff
@@ -0,0 +1,142 @@
+package ocs
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/models"
+	"github.com/openshift/assisted-service/pkg/conversions"
+)
+
+var _ = Describe("Ocs Operator Test GetHostRequirements", func() {
+	var (
+		ctx      = context.TODO()
+		operator = NewOcsOperator(common.GetTestLog())
+		master1  = &models.Host{Role: models.HostRoleMaster,
+			Inventory: Inventory(&InventoryResources{Cpus: 12, Ram: 32 * conversions.GB,
+				Disks: []*models.Disk{
+					{SizeBytes: 20 * conversions.GB, DriveType: "HDD"},
```

Comment:
Do these include the installation disk as well?

Reply:
the bootable disk? yes